### PR TITLE
[ACS-2240] Added test cases for moving files between sites and modify…

### PR DESCRIPTION
**Pull request by kcichonczyk**
Added test cases for:
-Moving document between sites
-Moving document to a site
-Moving document out of a site
-Document modification and removal

### CODE REVIEW

File: tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java [hunk 1]
```diff
@@ -215,6 +225,33 @@ public void testSiteUseCasesForCreateModifyDeleteSite()
         assertSiteQueryResult(testSite1.getId(), List.of());
         assertSiteQueryResult(ALL_SITES, List.of());
         assertSiteQueryResult(EVERYTHING, "AND", FILE_CONTENT_CONDITION, List.of(fileNotInSite));
+
+        Step.STEP("Moving files between sites and modifying content use cases");
```
Comment (SaraAspery) [inline tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java:229]: I guess the tests from this line onwards could either be moved to their own method. If not, then would you mind renaming the existing method?

File: tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java [hunk 2]
```diff
@@ -215,6 +225,33 @@ public void testSiteUseCasesForCreateModifyDeleteSite()
...
+
+        SiteModel publicSite1 = dataSite.usingUser(testUser).createSite(new SiteModel("public-site-1"));
+        SiteModel publicSite2 = dataSite.usingUser(testUser).createSite(new SiteModel("public-site-2"));
```
Comment (SaraAspery) [inline tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java:232]: there is a createSite method you could use if you wish to

**Review by SaraAspery — COMMENTED** [review]
LGTM! Just needs the new tests in a separate method, or the existing method renamed :)

File: tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java (hunk 1 above)
Comment (kcichonczyk) [reply tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java:229]: Do you mean that I should create a separate test method for my cases? or a method within your test method?

File: tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java (hunk 2 above)
Comment (kcichonczyk) [reply tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java:232]: Sure, I'll replace it

File: tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java [hunk 3]
```diff
@@ -215,6 +225,33 @@ public void testSiteUseCasesForCreateModifyDeleteSite()
...
+
+        //Moving a file between sites.
+        moveFile(file5, publicSite1, publicSite2);
+        assertSiteQueryResult(publicSite1.getId(), List.of(DOCUMENT_LIBRARY));
+        assertSiteQueryResult(publicSite2.getId(), List.of(DOCUMENT_LIBRARY, file5));
+
+        //Moving a file out of a site.
+        moveFileOutsideOfSite(file5, publicSite2, testFolder);
+        assertSiteQueryResult(publicSite2.getId(), List.of(DOCUMENT_LIBRARY));
+
```
Comment (pzhyland) [inline tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java:243]: Cover the scenario when a file is moved from out of any site to some site.

File: tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java (hunk 1 above)
Comment (SaraAspery) [reply tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java:229]: Either a separate test method for your cases, or rename the method from "testSiteUseCasesForCreateModifyDeleteSite()" to something that is more generic or covers document use cases as well.

**Review by SaraAspery — APPROVED** [review]
LGTM :)

File: tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java (hunk 3 above)
Comment (pzhyland) [reply tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java:250]: Test added

**Review by pzhyland — APPROVED** [review]

File: tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java (hunk 1 above)
Comment (kcichonczyk) [reply tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java:236]: moved to separate test method

File: tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java (hunk 2 above)
Comment (kcichonczyk) [reply tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/ElasticsearchSiteIndexingTests.java]: used as you suggested ;)

**Pull request merged**